src/components/ProductTours/AdminOnboardingTours/flows/AdministerSubscriptionsFlow.tsx

```diff
@@ -33,6 +36,10 @@ const AdministerSubscriptionsFlow = ({
   const isOnDetailPage = !!subscriptionUuid;
 
   const { isLoadingCustomerAgreement, subsidyRequestConfiguration } = useContext(SubsidyRequestsContext);
+  const { customerAgreement, isLoading: isLoadingAgreement } = useCustomerAgreement({ enterpriseId });
+  // @ts-ignore
```

Comment:
nit: It would be nice to have type information for `useCustomerAgreement`, but I can see that that part of the codebase hasn't had any TypeScript work done on it, so I'm fine leaving it for now.